web/src/components/molecules/Asset/Viewers/CsvViewer/Imagery.tsx

```diff
@@ -0,0 +1,80 @@
+import { Cartesian3 } from "cesium";
+import { useCallback, useEffect } from "react";
+import { useCesium } from "resium";
+
+import mapPin from "./mapPin.svg";
+
+type Props = {
+  url: string;
+};
+
+type GeoObj = {
+  lng?: string;
+  lat?: string;
+  [x: string]: string | undefined;
+};
+
+export const Imagery: React.FC<Props> = ({ url }) => {
+  const { viewer } = useCesium();
+
+  const dataFetch = useCallback(async () => {
+    const res = await fetch(url, {
+      method: "GET",
+    });
+    if (res.status !== 200) {
+      return;
+    }
+    return await res.text();
+  }, [url]);
+
+  const csvTextToObjects = useCallback((text: string) => {
+    const result = [];
+    const lines = text.split(/\r\n|\n|\r/);
+    const headers = lines[0].split(",");
+    for (let i = 1; i < lines.length; i++) {
+      const obj: GeoObj = {};
+      const line = lines[i].split(",");
+      for (let j = 0; j < headers.length; j++) {
+        obj[headers[j]] = line[j];
+      }
+      result.push(obj);
+    }
```

Comment:
Just a suggestion:
If you can use forEach or for ... of statements instead of a normal for loop, it would be better for code readability.